src/python/pants/help/help_printer.py

```diff
@@ -285,7 +366,7 @@ def _print_options_help(self, scope: str, show_advanced_and_deprecated: bool) ->
         for line in formatted_lines:
             print(line)
 
-    def _print_target_help(self, target_alias: str) -> None:
+    def _print_target_help(self, target_alias: str, show_advanced: bool) -> None:
```

Comment:
Unused?

Reply:
It is, but the method implements a interface, as all print help functions are invoked as `help_table[thing](thing, help_request.advanced)`, so perhaps we could mark the argument as unused in this case, but it must accept it, nonetheless.